recipes/mkdocs-click/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "mkdocs-click" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 00959d6cfcd9cf15af081f9f9a440027e84bff700fb4c14ba6e5efc0ac75ecf6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - click >=7, <8
+    - markdown >=3, <4
+
+test:
+  imports:
+    - mkdocs-click
+    - tests
```

Comment:
This test import feels odd?

Reply:
sure, not 100% sure how to do it, but it seems similar to this other recipe https://github.com/conda-forge/staged-recipes/pull/7963/files#diff-12a4d342475852c96556eec638bb5e4dcb178632670899694c4ade91afee2402R32

could we just remove it?

Reply:
Yeah it's a bug in both cases

Reply:
removed